shared/js/media/jpeg_metadata_parser.js

```diff
@@ -20,6 +20,10 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
   // This is the object we'll pass to metadataCallback
   var metadata = {};
 
+  // this is the start offset for the data in exif. Should be
+  // where to find the TIFF magic header.
+  var startOffset = 0;
```

Comment:
If this symbol is going to be used throughout, I'd like it to have a more specific name. startOffset is so generic. How about TIFFHeader or TIFFHeaderOffset.

Consider making it a const instead of a variable.  And please just initialize it to 10 instead of starting at 0 and incrementing later.